src/Domain/Model/User.php

```diff
@@ -34,6 +34,21 @@ public function meta(): HasMany
         return $this->hasMany(TalkMeta::class, 'admin_user_id');
     }
 
+    public function persistences(): HasMany
+    {
+        return $this->hasMany(Persistence::class, 'user_id');
+    }
+
+    public function reminders(): HasMany
+    {
+        return $this->hasMany(Reminder::class);
+    }
+
+    public function throttle(): HasMany
```

Comment:
For consistency, should we use plural here?

Reply:
The table is called throttle so I tried to keep it consistent...